src/js/weather.js

```diff
@@ -0,0 +1,34 @@
+import { DailyWeather } from './daily-weather';
+
+export class Weather {
+    constructor(location) {
+        this.location = location;
```

Comment:
you use instance of the whole class only for usage some data from it
very bad pattern

Reply:
fixed